page-tracking/src/page_info.rs

```diff
@@ -34,8 +34,8 @@ pub enum PageState {
     Shared(u64),
 
     /// Page is used to store hypervisor-internal state for the current owner, e.g. to back per-VM
-    /// data structures or as a page-table page for the VM.
-    VmState,
+    /// data structures or as a page-table page for the VM or hypervisor.
+    State,
```

Comment:
Unrelated, but typo (onwner) in commit message.

Reply:
Thanks!

Reply:
Should be fxied